### ISSUE TITLE
transaction errors from the daemon

### ISSUE THREAD

**Issue by ilsawa** [dashpay/p2pool-dash#63]
Have you tested the new version?
For some reason, there are transaction errors from the daemon.
At this time, all nodes lose contact with the daemon.
```
2019-02-05 06:13:26.061369 Outgoing connection to peer 195.114.96.40:8999 established. p2pool version: 1500 'f14af84-dirty'
2019-02-05 06:13:26.063215 Sending 1 shares to 195.114.96.40:8999
2019-02-05 06:13:26.162225 RECV remember_tx 00470200000001c2da0323f562e7e416bee073d6eeef1332683412c1034dacd5d45d6e152fc905010000006b483045022100bc5c91cf5199bba394d911fa57d5886f09e85485078fd61afc9690bb21e726f002201adbdcfd4dd6cb264f5ea87cddced694...
2019-02-05 06:13:26.163203 Lost peer 195.114.96.40:8999 - Connection was aborted locally, using.
2019-02-05 06:13:26.540497 > ########################################
2019-02-05 06:13:26.540659 > >>> Warning: LOST CONTACT WITH DASHD for 9.7 minutes! Check that it isn't frozen or dead!
2019-02-05 06:13:26.540729 > ########################################
2019-02-05 06:13:26.540794 P2Pool: 5257 shares in chain (4765 verified/5261 total) Peers: 7 (4 incoming)
2019-02-05 06:13:26.540848  Local: 73401MH/s in last 10.0 minutes Local dead on arrival: ~0.0% (0-5%) Expected time to share: 17.3 minutes
2019-02-05 06:13:26.540900  Shares: 81 (6 orphan, 3 dead) Stale rate: ~11.1% (5-20%) Efficiency: ~93.3% (84-99%) Current payout: (0.0071)=0.0071 DASH
2019-02-05 06:13:26.540959  Pool: 9525GH/s Stale rate: 4.8% Expected time to block: 16.3 hours
2019-02-05 06:13:26.748267 Outgoing connection to peer 94.156.218.28:8999 established. p2pool version: 1500 '1abf2e1-dirty'
2019-02-05 06:13:26.876021 Sending 1 shares to 94.156.218.28:8999
2019-02-05 06:13:26.878388 Incoming connection from peer 90.188.4.86:54164 established. p2pool version: 1500 'f14af84-dirty'
2019-02-05 06:13:26.904344 Sending 1 shares to 90.188.4.86:54164
2019-02-05 06:13:26.920724 Peer sent entire transaction 3a3e0b90f66ea4114886e4068e8290287d3ab3dbbcaaf5d02bfb12c730a6f698 that was already received
2019-02-05 06:13:26.930026 Lost peer 90.188.4.86:54164 - Connection to the other side was lost in a non-clean fashion.
2019-02-05 06:13:27.128538 Lost peer 94.156.218.28:8999 - Connection to the other side was lost in a non-clean fashion.
2019-02-05 06:13:27.534645 Outgoing connection to peer 185.128.12.21:8999 established. p2pool version: 1500 '18dc987-dirty'
2019-02-05 06:13:27.536478 Sending 1 shares to 185.128.12.21:8999
2019-02-05 06:13:27.810639 Lost peer 185.128.12.21:8999 - Connection to the other side was lost in a non-clean fashion.
2019-02-05 06:13:28.344948 Outgoing connection to peer 185.60.135.146:8999 established. p2pool version: 1500 'f14af84'
2019-02-05 06:13:28.346866 Sending 1 shares to 185.60.135.146:8999
2019-02-05 06:13:28.451348 RECV remember_tx 00440200000001c2da0323f562e7e416bee073d6eeef1332683412c1034dacd5d45d6e152fc905010000006b483045022100bc5c91cf5199bba394d911fa57d5886f09e85485078fd61afc9690bb21e726f002201adbdcfd4dd6cb264f5ea87cddced694...
2019-02-05 06:13:28.452445 Lost peer 185.60.135.146:8999 - Connection was aborted locally, using.
2019-02-05 06:13:29.542377 > ########################################
2019-02-05 06:13:29.542529 > >>> Warning: LOST CONTACT WITH DASHD for 9.7 minutes! Check that it isn't frozen or dead!
2019-02-05 06:13:29.542595 > ########################################

```

Node p2pool version: 222f57a-dirty
dashd   "version": 130000,  "protocolversion": 70213

**Comment by kr1z1s** [comment]
Traceback for this error

> Traceback (most recent call last):
  File "/usr/local/lib/pypy2.7/dist-packages/twisted/internet/tcp.py", line 249, in _dataReceived
    rval = self.protocol.dataReceived(data)
  File "/opt/p2pool-dash/p2pool/p2p.py", line 172, in new_dataReceived
    old_dataReceived(data)
  File "/opt/p2pool-dash/p2pool/util/p2protocol.py", line 27, in dataReceived
    self.dataReceived2(data)
  File "/opt/p2pool-dash/p2pool/util/datachunker.py", line 40, in _DataChunker
    wants = receiver.send(buf.get(wants))
--- <exception caught here> ---
  File "/opt/p2pool-dash/p2pool/util/p2protocol.py", line 57, in dataReceiver
    self.packetReceived(command, type_.unpack(payload, self.ignore_trailing_payload))
  File "/opt/p2pool-dash/p2pool/util/pack.py", line 63, in unpack
    obj = self._unpack(data, ignore_trailing)
  File "/opt/p2pool-dash/p2pool/util/pack.py", line 42, in _unpack
    obj, (data2, pos) = self.read((data, 0))
  File "/opt/p2pool-dash/p2pool/util/pack.py", line 295, in read
    item[key], file = type_.read(file)
  File "/opt/p2pool-dash/p2pool/util/pack.py", line 171, in read
    res[i], file = self.type.read(file)
  File "/opt/p2pool-dash/p2pool/util/pack.py", line 336, in read
    item[key], file = type_.read(file)
  File "/opt/p2pool-dash/p2pool/util/pack.py", line 171, in read
    res[i], file = self.type.read(file)
  File "/opt/p2pool-dash/p2pool/util/pack.py", line 295, in read
    item[key], file = type_.read(file)
  File "/opt/p2pool-dash/p2pool/util/pack.py", line 132, in read
    return read(file, length)
  File "/opt/p2pool-dash/p2pool/util/pack.py", line 16, in read
    raise EarlyEnd()
p2pool.util.pack.EarlyEnd:

**Comment by kr1z1s** [comment]
The problem persisted.
<img width="839" alt="2019-02-19 08 37 02" src="https://user-images.githubusercontent.com/5347595/52992473-bcf13400-3421-11e9-9d87-829e3cbbee0c.png">

**Comment by codablock** [comment]
Do you have debug.log files from affected dashd instances? Best would be if these instances would be run with "-debug=1"

**Comment by kr1z1s** [comment]
> Do you have debug.log files from affected dashd instances? Best would be if these instances would be run with "-debug=1"

This p2pool error. In dashd all goods

I own one of the big nodes.http://p2p-spb.xyz:7903/static/

`p2pool.util.pack.EarlyEnd:`

 Look at the hashrate chart. Error logs above.

**Comment by UdjinM6** [comment]
@ilsawa @kr1z1s pls try applying #67

**Comment by ilsawa** [comment]
I applied on the nodes:
http://crypto.mine.nu:7903
http://siberia.mine.nu:7903

**Comment by UdjinM6** [comment]
@ilsawa Looks like previous version didn't work as expected, I force-pushed a new one. Pls revert these changes and re-apply the new version.

**Comment by kr1z1s** [comment]
> @ilsawa Looks like previous version didn't work as expected, I force-pushed a new one. Pls revert these changes and re-apply the new version.

Add here - http://p2p-spb.xyz:7903/static/ 
Now

**Comment by ilsawa** [comment]
https://github.com/dashpay/p2pool-dash/pull/68 applied on
http://crypto.mine.nu:7903
http://siberia.mine.nu:7903

**Comment by UdjinM6** [comment]
>#68 applied on

@ilsawa just to make sure we understand each other: what I meant was "revert old #67 and reapply new #67". #68 does _not_ fix _this_ issue.

**Comment by ilsawa** [comment]
> @ilsawa just to make sure we understand each other: what I meant was "revert old #67 and reapply new #67". #68 does _not_ fix _this_ issue.

Thank you. At first I misunderstood you.
Already redid as it should.

**Comment by kr1z1s** [comment]
It works.

**Comment by UdjinM6** [comment]
Fixed by #67
